apps/framework-docs/src/pages/getting-started.mdx

```diff
@@ -1,9 +1,41 @@
-# Getting Started
-MooseJS is still in active development. We're working on stabilizing our APIs and adding more features. If you'd like to take it for a spin, we'd love to hear your feedback.
+# Introduction
+Welcome to the MooseJS documentation! MooseJS is still in early development, and we're talking to users and potential users all the time to make sure we're creating something valuable. If you're at all interested in what we're building here, then we'd love to talk to you! Please join our [Slack community](https://join.slack.com/t/igloocommunity/shared_invite/zt-25gsnx2x2-9ttVTt4L9LYFrRcM6jimcg) and reach out. We can't wait to hear from you!
 
-## Installation
-MooseJS is available on [npm](https://www.npmjs.com/package/moosejs). You can install it with the following command:
+---
 
-```txt filename="Terminal" copy
-npx create-moose-app my-app
-```
\ No newline at end of file
+## What is MooseJS?
+MooseJS is a JavaScript/Typescript framework for building data-intensive applications and services. Just like you use frontend application frameworks like NextJS to build your frontend applications, you can use MooseJS to build your data/analytics stack. MooseJS designed to be intuitive and scalable for software engineers who do not have a background in data engineering. You can combine MooseJS and your favorite frontend framework to build feature-rich, interactive data applications for your users and stakeholders.
+
+MooseJS is designed to be a batteries-included framework that helps you build your application without having to worry about the underlying data infrastructure. Under the hood, it's built with a Rust based core that helps you package, build and deploy your application to reliably take your apps from your local machine to production.
+
+## What types of applications can be built with MooseJS?
+MooseJS is great for any application that is based around a modern data/analytics stack. Any application where collecting data and extracting insights from that data is at the heart of the application. Common examples include:
+
+1. A data lake or data warehouse 
+2. An LLM-powered application
+3. An AI/ML model platform 
+3. A customer data analytics platform
+4. A monitoring / observability application 
```

Comment:
I would change the order - the perspective I have would be to sort by use-case easiest to get started with to harder.

1. A data lake or data warehouse 
2. A customer data analytics platform
4. A monitoring / observability application 
3. An LLM-powered application
5. An AI/ML model platform 

Also it looks like you have 2 3.

Reply:
good point. but going by hottest topic / most likely to interest the user instead.